data/tosca/yorc-slurm-types.yml

```
@@ -69,50 +117,30 @@ node_types:
   yorc.nodes.slurm.Job:
     derived_from: org.alien4cloud.nodes.Job
     properties:
-      name:
-        type: string
-        description: The slurm job name.
-        required: false
-      tasks:
-        description: Number of tasks to run.
-        type: integer
-        required: false
-        default: 1
-      nodes:
-        description: Number of nodes allocated to the job.
-        type: integer
-        required: false
-        default: 1
-      cpus_per_task:
-        description: Number of cpus allocated per task.
-        type: integer
+      job_properties:
```

```suggestion
      slurm_options:
```
As I understand it you grouped properties that are translated into Slurm options (I may be wrong) so I suggest to rename this property

---

right